docs/getting-started/README.mdx

```diff
@@ -2,53 +2,93 @@
 sidebar_position: 1
 title: Introduction
 slug: /
+pagination_next: getting-started/glossary
+hide_table_of_contents: true
 description: >-
   Getting started with Unlock Protocol for developers.
 ---
-
 import Admonition from "@theme/Admonition";
+import FlocklerEmbed from '@flockler/react-flockler-embed';
 
 <div className="container getting-started">
   <div className="row">
-    <div className="col col--12"></div>
+    <div className="col col--12">
+      <p>Unlock Protocol is an open-source, collectively owned, community-governed, 
+      peer-to-peer system that creates time-based memberships as ERC-721 NFTs. 
+      The system enables ERC-721s with additional functionality, unlocking 
+      decentralized, sustainable business models for the web like paid memberships, 
+      recurring automated subscriptions, credentialing, event ticketing, and more.
+      </p>
+    </div>
   </div>
   <div className="row">
     <div className="col col--6">
-      <a href="./getting-started/what-is-unlock">
-        <h3>Core Concepts</h3>
-      </a>
-      <p>
-        <strong>New to Unlock Protocol?</strong>
-      </p>
-      <p>Get familiar with some of our core concepts.</p>
+      <div className="card">
+        <div class="card__header">
+          <h3>Glossary of Terms</h3>
+        </div>
+        <div className="card__body">
+          <dl>
+            <dt>Lock</dt>
+            <dd>Customized smart contract generated from our factory/deployer contract.</dd>
```

Comment:
```suggestion
            <dd>Custom smart contract generated from our factory/deployer contract.</dd>
```

Reply:
> can/should we have titles and/or descriptions on the video tutorials? not sure what these two are
> 
> <img alt="Screen Shot 2022-11-11 at 5 02 22 PM" width="991" src="https://user-images.githubusercontent.com/214760/201449185-1529d67b-3368-4bce-9960-865abbd89bb2.png">

These are from a feed that is auto-updated from a playlist I created on our YouTube called "Code Tutorials". If we update the videos there with the proper cover image then it will update automatically here. If we continue to add anything new to that playlist then those new videos will auto-update here to keep this selection always showing the newest content first.

Reply:
for a "getting started" page, i think we want to feature specific (curated) videos, not just "the latest" from an auto-updated feed.

also think explicit copy with name / description / context "why?" is needed, above and beyond the youtube thumbnail.

recommend rethinking this section a bit.

Reply:
Added the video descriptions. Agreed that it needed to be different that's why I actually created a new playlist separate from the one we had called "Unlock Protocol Overview, Walkthroughs, and Tutorials" which was more of a mix of technical and non-technical explainer videos. I could additionally add a filter for a tag like "docs-landing" for that playlist so we could pair it down even more as that list grows. Right now we only have 6 that fit into that category.

Reply:
> what do you recommend?

Having thought about I think it's important enough that people see it straight at the top. Familiarity with the terms we use or even how we use specific terms is foundational to being able to understand the docs better.